### ISSUE TITLE
Add ddev & drainpipe

### ISSUE THREAD

**Issue by DBNubs** [DBNubs/onestophowtoguys#15]
# Todo

### Todo Details

Don't enable all the drainpipe featueres; drainpipe configuration will happen in other tasks.

From the root of the project run `ddev config`

Edit .ddev/config.yaml to match production environment

Don't commit all the ddev files. (the changes from ddev init we want, the things after ddev start we probs don't)

* Use same db as production (mysql or mariadb). [Manual step to download db is ok. Please note in Readme. Next task: Automate local set up will add drainpipe integration for downloading db automatically.]

* Use same webserver as production (pantheon nginx, acquia apache)

* If not already in the project, add stage_file_proxy code (composer require). Enable stage_file_proxy in ddev.

* Add a Readme if there isn't one, or update existing Readme

## Acceptance Criteria

A developer new to the project can:
* follow the steps in the readme
* have their local running with images and content, (OK if theme is not building perfectly, theme to be handled in separate task.)
* be able to log in with `ddev drush uli --uid=??`.
* composer.json has drainpipe (in require) and drainpipe-dev (in require-dev)

## Context
* https://architecture.lullabot.com/adr/20220920-git-branch-naming/
* https://architecture.lullabot.com/adr/20211207-ddev-locals/ 
* https://architecture.lullabot.com/adr/20210729-stage-file-proxy/  
* https://architecture.lullabot.com/adr/20211026-use-settings-not-splits/ 

## Example

TBD

## Timebox

Timebox 6 hours. Make a comment with an WIP update after each half day (3 hours). Please ping the PM when reach the end of the first timebox to discuss adding next timebox.

## Timeline

This unblocks a bunch of other tasks, goal is to complete in the first couple weeks of the project.

## Merge Target

master/main"
